### PR TITLE
modify the variable name specification to Camel-Case

### PR DESCRIPTION
**Pull request by doublewinter0**
Signed-off-by: doublewinter0 <erdong.me@gmail.com>

### Context

**In order to have a better reading experience, I modify the variable name specification to Camel-Case.**

### Contributor Checklist
- [x] [Review Contribution Guidelines](https://github.com/gradle/gradle/blob/master/CONTRIBUTING.md)
- [x] Make sure that all commits are [signed off](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff) to indicate that you agree to the terms of [Developer Certificate of Origin](https://developercertificate.org/).
- [x] Make sure all contributed code can be distributed under the terms of the [Apache License 2.0](https://github.com/gradle/gradle/blob/master/LICENSE), e.g. the code was written by yourself or the original code is licensed under [a license compatible to Apache License 2.0](https://apache.org/legal/resolved.html).
- [x] Check ["Allow edit from maintainers" option](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) in pull request so that additional changes can be pushed by Gradle team
- [ ] Provide integration tests (under `<subproject>/src/integTest`) to verify changes from a user perspective
- [ ] Provide unit tests (under `<subproject>/src/test`) to verify logic
- [ ] Update User Guide, DSL Reference, and Javadoc for public-facing changes
- [x] Ensure that tests pass sanity check: `./gradlew sanityCheck`
- [x] Ensure that tests pass locally: `./gradlew <changed-subproject>:quickTest`

### Gradle Core Team Checklist
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation
- [ ] Recognize contributor in release notes

### CODE REVIEW

**Comment by doublewinter0** [comment]
@bot-gradle 
This is the first time I have contributed code to gradle. I sincerely hope to give me some comments. Regardless of whether it can be merged or not, I will be very grateful, and I will still contribute to the gradle community in the future, thank you!

**Comment by blindpirate** [comment]
@doublewinter0 , `@bot-gradle` is an automated tool for internal usage only, thus it won't respond to non-gradle-member comments. You can just ignore it.

**Comment by doublewinter0** [comment]
> 
> Sorry but I'm afraid you're not allowed to do this.

I want to know if there is a problem with the way I requested the merge or the code I submitted does not make sense to merge? Can you give me some suggestions? Thank you!

**Comment by blindpirate** [comment]
@bot-gradle test RFN

File: subprojects/core/src/main/java/org/gradle/groovy/scripts/DefaultScript.java [hunk 1]
```diff
@@ -68,12 +68,12 @@
     private ProviderFactory providerFactory;
     private LoggingManager loggingManager;
 
-    public ServiceRegistry __scriptServices;
+    public ServiceRegistry scriptServices;
```
Comment (blindpirate) [inline subprojects/core/src/main/java/org/gradle/groovy/scripts/DefaultScript.java:71]:
```suggestion
    private ServiceRegistry scriptServices;
```

Let's also make it private. It seems not referenced anywhere. Most likely to be legacy code. Thanks for catching it!

**Comment by blindpirate** [comment]
@bot-gradle test this

**Comment by blindpirate** [comment]
@bot-gradle test and merge

**Pull request merged**